salt/vault/service.sls

```diff
@@ -0,0 +1,25 @@
+{% from "vault/map.jinja" import vault with context %}
+
+manage_selinux_mode:
+  selinux.mode:
+    - name: permissive
+
+vault_service_init_file_managed:
+  file.managed:
+    - name: {{ vault.service.path }}
+    - source: {{ vault.service.source }}
+    - template: jinja
+    - defaults:
+{%- if vault.dev_mode %}
+        config: -dev -dev-root-token-id=root -config /srv/salt/vault/files/server.dev.hcl
```

Comment:
i don't see another reference in the pr to `server.dev.hcl`? am i missing it? or where does this come from?

Reply:
You are correct. it's more for local dev with vagrant. I will include the vagrant file as part of the project.